network-api/networkapi/wagtailpages/pagemodels/products.py

```diff
@@ -1232,9 +1232,10 @@ def categories_page(self, request, slug):
 
         context['category'] = category.slug
         context['products'] = products
-        context['pageTitle'] = pgettext(
+        context['pageTitle'] = f'{category} | ' + pgettext(
                 'This can be localized. This is a reference to the “*batteries not included” mention on toys.',
-                '*privacy not included') + f' - {category}'
+                'Privacy & Security Guide'
+            ) + f' | Mozilla Foundation'
```

Comment:
same issue as mentioned in https://github.com/mozilla/foundation.mozilla.org/pull/6143/files#r572438582